scrape_data/cta_schedule_versions.py

```diff
@@ -0,0 +1,59 @@
+import boto3
+import sys
+import data_analysis.static_gtfs_analysis as sga
+import pendulum
+from io import StringIO
+
+ACCESS_KEY = sys.argv[1]
+SECRET_KEY = sys.argv[2]
+
+client = boto3.client(
+    's3',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_KEY
+)
+
+s3 = boto3.resource(
+    's3',
+    region_name='us-east-1',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_KEY
+)
+
+date = pendulum.now().to_date_string()
+
+zipfile, zipfile_bytes_io = sga.download_cta_zip()
+print(f'Saving zipfile available at '
+      f'https://www.transitchicago.com/downloads/sch_data/google_transit.zip '
+      f'on {date} to public bucket')
+zipfile_bytes_io.seek(0)
+client.upload_fileobj(zipfile_bytes_io, 'chn-ghost-buses-public', f'google_transit_{date}.zip')
```

Comment:
Can we add another directory level in this path? Right now the zipfiles are just being written to the root directory which doesn't pose a technical problem but will probably get a bit messy quickly. Maybe something like `f'cta_schedule_zipfiles_raw/google_transit_{date}.zip`?

Reply:
Good point, I can add a new directory.